docs/data/sql_functions.yml

```
@@ -617,6 +617,9 @@ collection:
   - sql: ARRAY_DISTINCT(haystack)
     table: haystack.arrayDistinct()
     description: Returns an array with unique elements. If the array itself is null, the function will return null. Keeps ordering of elements.
+  - sql: ARRAY_REMOVE(haystack, needle)
+    table: haystack.arrayRemove(needle)
+    description: Remove all elements that equal to element from array. If the array itself is null, the function will return null. Keeps ordering of elements.
```

Sorry, looks like missed it 
```suggestion
    description: Removes all elements that equal to element from array. If the array itself is null, the function will return null. Keeps ordering of elements.
```

---

have fixed